src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -77,6 +76,7 @@ export interface MembershipConfig {
      * Use the new Manager.
      *
      * Default: `false`.
+     * @deprecated does nothing anymore we always default to the new memberhip manager.
```

Comment:
I see, so this isn't even a breaking change as we're not exporting the membership managers to begin with. Nice :+1: